terms.rdf

```diff
@@ -255,6 +315,34 @@
         <dc:hasVersion rdf:resource="http://cldf.clld.org/v1.1"/>
     </rdf:Property>
 
+    <rdf:Property rdf:about="http://cldf.clld.org/v1.0/terms.rdf#speakerArea"
+                  dc:type="reference-property">
+        <rdfs:comment rdf:parseType="Literal">
+            <p>
+                An identifier referencing a media resource by providing a foreign key into MediaTable.
```

Comment:
Again, not sure about the preposition. What I've written above might just be a wrong suggestion, so feel free to ignore.